files/jsondata/GroupData.json

```
@@ -1967,6 +1967,7 @@
         "/docs/Web/API/WebRTC_API/Signaling_and_video_calling",
         "/docs/Web/API/WebRTC_API/Using_data_channels",
         "/docs/Web/API/WebRTC_API/Using_DTMF",
+        "/docs/Web/API/Analyzing_WebRTC_statistics",
         "/docs/Web/API/WebRTC_API/adapter.js",
```

Removing the Analyzing docs - we don't need it in current form.
```suggestion
        "/docs/Web/API/WebRTC_API/Using_DTMF",
        "/docs/Web/API/WebRTC_API/adapter.js",
```